app/cdap/components/DirectiveInput/Components/DirectiveUsage/index.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, Divider, Typography, Link } from '@material-ui/core';
+import { moreInfoOnDirective, PREFIX } from 'components/DirectiveInput/constants';
+import { IDirectiveUsageProps } from 'components/DirectiveInput/types';
+import T from 'i18n-react';
+import React from 'react';
+import styled from 'styled-components';
+import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
+
+const DirectiveWrapper = styled(Box)`
+  padding: 10px;
+`;
+
+const UsageText = styled(Typography)`
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  letter-spacing: 0.15;
+  color: #ffffff;
+  margin-bottom: 10px;
+  display: flex;
+  align-items: center;
+  width: 100%;
+`;
+
+const DividerLine = styled(Divider)`
+  &.MuiDivider-root {
+    background-color: #ffffff;
+  }
+`;
+
+const InfoLink = styled(Link)`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #79b7ff;
```

Comment:
plz remove the linter in ur vs code and change them to capital letter, this is important